src/main/java/org/truffleruby/core/numeric/FloatNodes.java

```diff
@@ -138,9 +138,7 @@ protected Object mulCoerced(double a, Object b,
 
         private final ConditionProfile complexProfile = ConditionProfile.createBinaryProfile();
 
-        @Specialization(
-                guards = { "exponent == cachedExponent", "cachedExponent >= 0", "cachedExponent < 10" },
-                limit = "10")
+        @Specialization(guards = { "exponent == cachedExponent", "cachedExponent >= 0", "cachedExponent < 10" }, limit = "10")
```

Comment:
The recommended config for reformat got me to this result. Does this also happens in your editors?

Reply:
No, I guess you need `mx intellijinit` and/or `jt format`, see
https://github.com/oracle/truffleruby/commit/771810db50b30d826c87df337e25f4f0c5e71e54#commitcomment-35763709
In case of doubt, `jt format` does the right thing.
Once the IDE is properly configured, it should be the same in the IDE too.

Reply:
Note that this might be due to an older `mx` that increased the wrapping to 200 or so, updating `mx` should help for that. cc @pitr-ch

Reply:
To be able to merge this quickly, could you remove the reformat commit, and I'll just reformat on top for this PR?

Reply:
Reformatted the code to remove this change.

Reply:
Thanks, the formatter check is happy now.